plugins/docker_binary_builder/test/models/binary_builder_test.rb

```diff
@@ -42,6 +46,23 @@
         "Cleaning up docker build image and container...\n"].join
     end
 
+    it 'run pre build shell script if it is available' do
+      File.expects(:exists?).with(File.join(dir, BinaryBuilder::DOCKER_BUILD_FILE)).returns(true)
+      File.expects(:exists?).with(File.join(dir, BinaryBuilder::PRE_BUILD_SCRIPT)).returns(true)
+      executor.expects(:execute!).with(File.join(dir, BinaryBuilder::PRE_BUILD_SCRIPT)).once
+
+      project.update_attributes(deploy_with_docker: true)
+      builder.build
+      output.string.must_equal [
+                                 "Running pre build script...\n",
```

Comment:
2-space indent plz ... wastest less space and does not need to all be changed when first line changes